src/components/bodyImage.tsx

```diff
@@ -56,16 +56,20 @@ const captionStyles = css`
 const BodyImage: FC<Props> = ({ figcaption, image, pillar }: Props) =>
     <figure css={styles}>
         <Image {...image} sizes={sizes} />
-        <figcaption css={captionStyles}>
-            <svg
-                css={triangleStyles(getPillarStyles(pillar))}
-                viewBox="0 0 10 9"
-                xmlns="http://www.w3.org/2000/svg"
-            >
-                <polygon points="0,9 5,0 10,9 0,9" />
-            </svg>
-            {figcaption}
-        </figcaption>
+        {
+            figcaption.fmap(caption =>
+                <figcaption css={captionStyles}>
+                    <svg
+                        css={triangleStyles(getPillarStyles(pillar))}
+                        viewBox="0 0 10 9"
+                        xmlns="http://www.w3.org/2000/svg"
+                    >
+                        <polygon points="0,9 5,0 10,9 0,9" />
+                    </svg>
+                    {caption}
+                </figcaption>
+            )
```

Comment:
I think this might be why validation is failing:

```suggestion
            ).withDefault(null)
```